ironside/include/ironside/se/uicr.h

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IRONSIDE_SE_UICR_H_
+#define IRONSIDE_SE_UICR_H_
+
+#include <stdint.h>
+
+#include <ironside/se/internal/mdk.h>
```

Comment:
Is this actually needed here? It looks a bit weird that this header includes back `ironside/se/uicr.h`.